### PR TITLE
update Tidying_Data_with_tidyr

### PR DESCRIPTION
**Pull request by wenliwyan**

### CODE REVIEW

File: Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/lesson.yaml [hunk 1]
```diff
@@ -227,7 +227,7 @@
   Hint: Call bind_rows() with two arguments, passed and failed (in that order), to join the two tables.
 
 - Class: text
-  Output: Of course, we could arrange the rows however we wish at this point, but the important thing is that each row is an observation, each column is a variable, and the table contains a single observational unit. Thus, the data are tidy.
+  Output: Of course, we could arrange the rows as we wish at this point, but the important thing is that each row is an observation, each column is a variable, and the table contains a single observational unit. Thus, the data are tidy.
```
Comment (seankross) [inline Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/lesson.yaml]: I don't see how changing `however` to `as` clarifies this statement. I like your other changes.

**Comment by ncarchedi** [comment]
@wenliwyan Can you also update script8-correct.R, which is identical to the script you corrected, except with the correct answer filled in?

File: Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/lesson.yaml (hunk 1 above)
Comment (wenliwyan) [inline Getting_and_Cleaning_Data/Tidying_Data_with_tidyr/lesson.yaml]: I have changed `however` back. Sorry that English is not my mother language, and I misunderstood the however here as "although". :P

**Comment by seankross** [comment]
Thanks!

**Pull request merged**